doc/langref/test_union_method.zig

```diff
@@ -18,11 +18,13 @@ const Variant = union(enum) {
 };
 
 test "union method" {
-    var v1 = Variant{ .int = 1 };
-    var v2 = Variant{ .boolean = false };
+    var v1: Variant = .{ .int = 1 };
+    var v2: Variant = .{ .boolean = false };
+    var v3: Variant = .none;
```

Comment:
not hugely important, but might be nice to note that since the payload type is (implicit) `void`, initialization with an explicit void value is exactly equivalent: `var v3b: Variant = .{ .none = {} };`